tasks/main.yml

```diff
@@ -1,2 +1,19 @@
 ---
-# tasks file for skeleton
+# tasks file for vdp-scanner
+- name: Create the /var/cyhy/vdp directory
+  ansible.builtin.file:
+    path: /var/cyhy/vdp
+    state: directory
+    mode: 0755
+
+- name: Copy Docker Compose file
+  ansible.builtin.copy:
+    src: docker-compose.yml
+    dest: /var/cyhy/vdp
+    mode: 0644
+
+- name: Create the mount point for output
+  ansible.builtin.file:
+    path: /var/cyhy/vdp/output
+    state: directory
+    mode: 0755
```

Comment:
The keys under the tasks here are not alphabetized.  We have been trying to do that of late.

Reply:
Ah, I started this a while ago and forgot to take that into account when I revisited this repo. Fixed in 1c70461. Thanks for the catch!